shared/actions/json/wallets.json

```diff
@@ -304,6 +308,13 @@
       "_description": "Set whether the payment is ready to send",
       "readyToSend": "boolean"
     },
+    "startPayment": {
```

Comment:
@buoyad Hm, I think it gets racy -- the first thing the openSendRequestForm saga does is clear out `state.wallets.building`, and that's where we want to record the new `bid`, so we'd have to avoid that somehow.  The current code uses Saga.sequentially to define that ordering.  What do you think?

Reply:
Ah hm, good point. We could use Miles' approach on this case too, if the reducer clears building on `openSendRequestForm`, I think the race would be gone.

Reply:
@buoyad Nice idea, thanks -- how's this look?